view/code/index.pug

```diff
@@ -6,7 +6,7 @@
 .code.is-rgb
   +field('hex/rgba CSS code', 'r', {
     type: 'text',
-    options: {
+    'data-options': {
```

Comment:
It is better to fix `+field` mixin and avoid passing `options` and `optionsLabel` to `<input>`

https://github.com/evilmartians/oklch-picker/blob/main/view/field/mixin.pug